### DISCUSSION TITLE
`RANK` data

### DISCUSSION THREAD

**Discussion by Twistace** [kyleskom/NBA-Machine-Learning-Sports-Betting#440] [General]
Hello guys!

I came across this project and took a quick look at the dataset used for the models. Tifferent statistical characteristics are scrapped but couldn't figure out what does the `rank` extensions stand for. 

Most commonly in this field (game predicting) I see rolling avareges and moving the data -1 on y axis (in respect to the target to avoid data leakage), but not this rank feature. What does it indicate?

**Comment by kyleskom** [comment]
Rank columns are that teams rank for that stats against other teams.
